cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/sponge/platform/CloudNetSpongeStatusClient.java

```diff
@@ -0,0 +1,32 @@
+package de.dytanic.cloudnet.ext.bridge.sponge.platform;
+
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.api.MinecraftVersion;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.network.status.StatusClient;
+
+import java.net.InetSocketAddress;
+import java.util.Optional;
+
+class CloudNetSpongeStatusClient implements StatusClient {
+
+    static final StatusClient INSTANCE = new CloudNetSpongeStatusClient();
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5823302)